app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,258 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/background_Color">
+    <ImageView
+        android:id="@+id/img_background"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="30dp"
+        android:src="@drawable/orig"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:background="@drawable/gradient"
+        app:layout_constraintStart_toStartOf="@id/img_background"
+        app:layout_constraintEnd_toEndOf="@id/img_background"
+        app:layout_constraintTop_toTopOf="@id/img_background"
+        app:layout_constraintBottom_toBottomOf="@id/img_background"/>
+    <TextView
+        android:id="@+id/back"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="60dp"
+        android:drawableStart="@drawable/ic_path"
+        android:drawablePadding="8dp"
+        android:text="@string/back_text_View"
+        android:textColor="@color/for_Back"
+        android:textSize="12dp"
+        app:layout_constraintStart_toStartOf="@id/img_background"
+        app:layout_constraintTop_toTopOf="@id/img_background"/>
+    <TextView
+        android:id="@+id/age_Limit_Text_View"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/age_limit"
+        android:textColor="@color/white"
+        android:textSize="12dp"
+        android:textStyle="bold"
+        android:layout_marginTop="4dp"
+        app:layout_constraintStart_toStartOf="@id/img_background"
```

Comment:
think about bigger name for movie
maybe better relate limit depending on name, but not vice verse